test/functional/android/helper/test_helper.py

```diff
@@ -36,11 +32,16 @@
 class BaseTestCase:
     def setup_method(self, method) -> None:  # type: ignore
         desired_caps = desired_capabilities.get_desired_capabilities('ApiDemos-debug.apk.zip')
-        self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
+        self.driver = webdriver.Remote(
+            'http://localhost:4723/wd/hub', options=UiAutomator2Options().load_capabilities(desired_caps)
```

Comment:
Seems good, overall.

One confirmation is... when a user wants to set an arbitrary automation name etc (not defined in this client one), they can use:

```
options = AppiumOptions()
options.automation_name('arbitrary name').set_capability('ownCapability', 'the value')...
...

 self.driver = webdriver.Remote('http://localhost:4723/wd/hub', options=options)
```

, correct?

Reply:
yes, that's correct

Reply:
small correction to the above code:

```
options = AppiumOptions()
options.automation_name = 'arbitrary name'
options.set_capability('ownCapability', 'the value')\
    .set_capability('appium:ownCapability2', 'the value')
     ....
```

Reply:
thank you